engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -223,10 +223,10 @@ class WorkflowManagerActor(config: Config, val workflowStore: ActorRef)
     * Submit the workflow and return an updated copy of the state data reflecting the addition of a
     * Workflow ID -> WorkflowActorRef entry.
     */
-  private def submitWorkflow(workflow: WorkflowStore.WorkflowToStart): WorkflowIdToActorRef = {
+  private def submitWorkflow(workflow: WorkflowStoreDatabase.WorkflowToStart): WorkflowIdToActorRef = {
     val workflowId = workflow.id
 
-    val startMode = if (workflow.state == WorkflowStore.Restartable) {
+    val startMode = if (workflow.state == WorkflowStoreDatabase.Restartable) {
```

Comment:
Hah, you barely dodged my original scathing reply. My machine rebooted as I was about to hit 'comment'